pkg/model/storage_spec.go

```diff
@@ -28,6 +28,9 @@ type StorageSpec struct {
 	// The path of the host data if we are using local directory paths
 	SourcePath string `json:"SourcePath,omitempty"`
 
+	// Allow write access for locally mounted inputs
+	ReadWrite bool `json:"ReadWrite,omitempty"`
```

Comment:
Which `StorageSourceType` does this field apply to? Could you leave a comment stating this? It will make my rebase of the `StorageSpec` changes easier.